### PR TITLE
Adding rules button into lower nav bar and the icon. 

### PR DESCRIPTION
**Pull request by michalhus**
Added rules component to Avalon. 

###  Need to update rule text for Avalon in the Rule component.

### CODE REVIEW

File: src/games/avalon/views/lobby.tsx [hunk 1]
```diff
@@ -110,6 +117,17 @@ export const LobbyAvalon: React.SFC<Props> = ({ startGame, lobby }) => {
             </Badge>
           }
         />
+                
+        <BottomNavigationAction
+          label="Rules"
+          value={View.rules}
+          icon={
+            <Badge badgeContent={lobby.avalonRoles.length}>
```
Comment (kaw2k) [inline src/games/avalon/views/lobby.tsx]: Let's remove the badge here

File: src/games/avalon/components/rules.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      
+      <Typography>
+        If Lady of The Lake is in play then player to the right of the 1st Quest leader 
+        is to claim it. After 2nd Quest, player that claimed Lady of The Lake can inspect 
+        other players to see if they are Good or Evil. The player that was inspected now 
+        is possesion of the Lady of The Lake and can inspected someone after next Quest.
+        Lady of The Lake is destroyed after person that already claimed her is inspected.
+      </Typography>
+
+      <ActionRow className={classes.padded} fixed>
```
Comment (kaw2k) [inline src/games/avalon/components/rules.tsx]: Let's remove this action row, we can use the bottom navigation to navigate back

File: src/games/avalon/components/rules.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
+import * as React from 'react'
+import { Typography } from '@material-ui/core'
+import { ActionRow } from '../../../components/actionRow'
+import { Button } from '../../../components/button'
+import { makeStyles } from '@material-ui/styles'
+
+interface Props {
+  done: () => void
```
Comment (kaw2k) [inline src/games/avalon/components/rules.tsx]: Let's also remove done here

File: src/games/avalon/views/lobby.tsx [hunk 4]
```diff
@@ -69,6 +71,11 @@ export const LobbyAvalon: React.SFC<Props> = ({ startGame, lobby }) => {
 
       {view === View.lobby && <Players lobby={lobby} />}
       {view === View.roles && <Roles lobby={lobby} />}
+      {view === View.rules && 
+        <Layout padded>
```
Comment (kaw2k) [inline src/games/avalon/views/lobby.tsx]: Let's also remove layout here

File: src/games/avalon/components/rules.tsx (hunk 2 above)
Comment (michalhus) [reply src/games/avalon/components/rules.tsx]: Done

File: src/games/avalon/views/lobby.tsx (hunk 1 above)
Comment (michalhus) [reply src/games/avalon/views/lobby.tsx]: Done

File: src/games/avalon/components/rules.tsx (hunk 3 above)
Comment (michalhus) [reply src/games/avalon/components/rules.tsx]: Done.

Had to remove some extra code in the component to make this work

File: src/games/avalon/views/lobby.tsx (hunk 4 above)
Comment (michalhus) [reply src/games/avalon/views/lobby.tsx]: Done

**Pull request merged**